src/fhirtypes/ElementDefinition.ts

```diff
@@ -1520,12 +1520,22 @@ export class ElementDefinition {
    * @throws {ValueAlreadyAssignedError} when the code is already assigned to a different code
    * @throws {InvalidUriError} when the system being assigned is not a valid uri
    */
-  private assignFshCode(code: FshCode, exactly = false): void {
+  private assignFshCode(code: FshCode, exactly = false, fisher?: Fishable): void {
```

Comment:
Since `exactly` is an optional argument, could we keep it last? If we put `fisher` last, we will then always have to specify `exactly`.

Reply:
Hmmm...  So I think I might disagree with @ngfreiter (!!).  While I agree that optional arguments should be last, I'd also note that `fisher` is an optional argument, so...

I actually think `fisher` should go last in order to maintain consistency between function signatures.  For example, the primary (public) `assignValue` signature takes these same three arguments and has `fisher` _last_:
```ts
assignValue(value: AssignmentValueType, exactly = false, fisher?: Fishable): void
```

The other (private) assign functions take the `exactly` argument after the `value`:
```ts
assignFHIRValue(fshValue: string, fhirValue: any, exactly: boolean, type: string)
assignNumber(value: number, exactly = false): void
assignString(value: string, exactly = false): void
```

So, to maintain consistency with `assignValue` (and to an extent, the others), I'd suggest reverting back to @julianxcarter's original signature:
```ts
private assignFshCode(code: FshCode, exactly = false, fisher?: Fishable): void
```

Reply:
Ohhh that's my bad. @cmoesel is definitely right here. Sorry for making you change that @julianxcarter!